sched_b_baseline/sched_b.list

```diff
@@ -0,0 +1,18 @@
+>cycle141618_cmte_tp_w
+https://api.open.fec.gov/v1/schedules/schedule_b/?spender_committee_type=W&sort=-disbursement_date&per_page=20&sort_null_only=false&sort_hide_null=false&api_key=DEMO_KEY&two_year_transaction_period=2018&two_year_transaction_period=2016&two_year_transaction_period=2014
+>cycle141618_purpose_cat_other
+https://api.open.fec.gov/v1/schedules/schedule_b/?sort=-disbursement_date&per_page=20&sort_null_only=false&disbursement_purpose_category=other&sort_hide_null=false&api_key=DEMO_KEY&two_year_transaction_period=2018&two_year_transaction_period=2016&two_year_transaction_period=2014
+>cycle141618_line_num_23
+https://api.open.fec.gov/v1/schedules/schedule_b/?sort_hide_null=false&per_page=20&two_year_transaction_period=2018&two_year_transaction_period=2016&two_year_transaction_period=2014&sort=-disbursement_date&api_key=DEMO_KEY&line_number=23&sort_null_only=false
+>cycle141618_desp_contribution2
+https://api.open.fec.gov/v1/schedules/schedule_b/?sort_hide_null=false&per_page=20&two_year_transaction_period=2018&two_year_transaction_period=2016&two_year_transaction_period=2014&sort=-disbursement_date&api_key=DEMO_KEY&sort_null_only=false&disbursement_description=contribution:2 earmark:1
+>cycle141618_cmte_id_actblue
+https://api.open.fec.gov/v1/schedules/schedule_b/?sort_hide_null=false&per_page=20&two_year_transaction_period=2018&two_year_transaction_period=2016&two_year_transaction_period=2014&sort=-disbursement_date&committee_id=C00401224&api_key=DEMO_KEY&sort_null_only=false
+>cycel141618_recipient_st
+https://api.open.fec.gov/v1/schedules/schedule_b/?sort_hide_null=false&per_page=20&two_year_transaction_period=2018&two_year_transaction_period=2016&two_year_transaction_period=2014&sort=-disbursement_date&api_key=DEMO_KEY&sort_null_only=false&recipient_state=DC
+>cycle_141618_recipient_city
+https://api.open.fec.gov/v1/schedules/schedule_b/?sort_hide_null=false&per_page=20&two_year_transaction_period=2018&two_year_transaction_period=2016&two_year_transaction_period=2014&sort=-disbursement_date&api_key=DEMO_KEY&sort_null_only=false&recipient_city=washington
+>cycle141618_top_recipent_name
+https://api.open.fec.gov/v1/schedules/schedule_b/?sort_hide_null=false&per_page=20&two_year_transaction_period=2018&two_year_transaction_period=2016&two_year_transaction_period=2014&sort=-disbursement_date&api_key=DEMO_KEY&sort_null_only=false&recipient_name=democratic congressional campaign 
+>cycle141618_combine
+https://api.open.fec.gov/v1/schedules/schedule_b/?spender_committee_type=W&sort=-disbursement_date&line_number=23&per_page=20&sort_null_only=false&disbursement_purpose_category=other&sort_hide_null=false&api_key=DEMO_KEY&committee_id=C00401224&two_year_transaction_period=2018&two_year_transaction_period=2016&two_year_transaction_period=2014&disbursement_description=travel
```

Comment:
@qqss88 I don't believe `line_number=23` exists for schedule_b. Do you mean `line_number=F3X-23`?